vendor/github.com/BurntSushi/toml/.travis.yml

```diff
@@ -2,11 +2,14 @@ language: go
 go:
   - 1.1
   - 1.2
+  - 1.3
+  - 1.4
+  - 1.5
+  - 1.6
```

Comment:
What happens when you provide multiple go versions here? Is there a build run with every version each? I would say we only run using one version, or at least the 3 most recent go versions.